Server setting backend

ez4papiz1 opened the pull request:
Created 2 files Server.model.js and ServerSettings.js to handle the backend routing for server settings

jordanp2002 commented on DiscordCode/BackEnd/routes/ServerSettings.js:
```diff
@@ -0,0 +1,53 @@
+const mongoose = require('mongoose');
+const Server = require('./models/Server');
+
+mongoose.connect('mongodb://localhost:27017/colligo');
```
should be createConnection not .connect

jordanp2002 commented on DiscordCode/BackEnd/routes/ServerSettings.js:
```diff
@@ -0,0 +1,53 @@
+    }
+});
+
+router.post('/banUser', async (req, res) => {
```
should be return some information via json so frontend can display this information to the user. Right now we a just sending status messages.

jordanp2002 commented on DiscordCode/BackEnd/routes/Models/Server.model.js:
```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema
+
+const UserSchema = new Schema({
```
ServerModel has already been created and implemented for the ServerPage, maybe this could be turned into a strictly Server Settings model with sid being a key for both the server and server settings

ez4papiz1 commented:
-deleted redundant Server.Model.js model file
-fixed ServerSetting.js with ServerData.js model file
-all route files now send information back via JSON
-use .createConnection instead of .connect
